akka-protobuf/src/main/java/akka/protobuf/RpcChannel.java

```diff
@@ -4,7 +4,7 @@
 
 // Protocol Buffers - Google's data interchange format
 // Copyright 2008 Google Inc.  All rights reserved.
-// http://code.google.com/p/protobuf/
+// https://github.com/protocolbuffers/protobuf
```

Comment:
I'm not sure we should change URL's in copyright headers... if we do update it, I think we should follow upstream (https://github.com/protocolbuffers/protobuf/blob/master/java/core/src/main/java/com/google/protobuf/RpcChannel.java) and change it to https://developers.google.com/protocol-buffers/

Reply:
Yeah, you're right. Let's stick to the sourced files.

Reply:
@GKuldeepak-Knoldus could you revert the changes to the copyright headers, pls

Reply:
@ennru , I have reverted the changes as suggested by you. Please have a look.